docs/modules/ROOT/pages/how-tos/add-servala-control-cluster.adoc

```diff
@@ -0,0 +1,162 @@
+= Add a Servala Control Cluster
+
+Currently service clusters run as vclusters on one of the CSP's service clusters.
+Usually it runs on the first provisioned cluster.
```

Comment:
```suggestion
The Servala Control Cluster (aka control-plane) runs in a vcluster on one of the CSP's service clusters, usually on the first provisioned service cluster.
```

I'd refrain from using "currently" in such documentation. This always implies some uncertainty. Document the fact, plans belong in Jira.